### PR TITLE
refactor: Enable triage process to run `@serverless/compose`

### PR DESCRIPTION
**Pull request by pgrzesik**
Internal

### CODE REVIEW

File: lib/cli/triage.js [hunk 1]
```diff
@@ -47,6 +47,23 @@ module.exports = async () => {
...
+            await fsp.access(`serverless-compose.${extension}`);
+            return true;
+          } catch {
+            return false;
+          }
+        })
+      )
+    ).some(Boolean)
+  ) {
+    return '@serverless/compose';
```
Comment (medikoo) [inline lib/cli/triage.js:64]: I would follow convention, we have so far, where CLI name matches package name

File: test/unit/lib/cli/triage/index.test.js [hunk 2]
```diff
@@ -95,5 +95,11 @@ describe('test/unit/lib/cli/triage/index.test.js', () => {
         register('@serverless/components', 'us');
       }
     }
+
+    it('Should recognize "@serverless/compose" if "serverless-compose.yml" file present', async () => {
```
Comment (medikoo) [inline test/unit/lib/cli/triage/index.test.js:99]: I think it's enough to add CLI name to L60, and it then should be automatically tested (no need for other `it`)

File: lib/cli/run-compose.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+              'Could not create "package.json" in current directory.',
+              'Please create it manually and run this command again.',
+            ].join('\n')}\n`
+          );
+          return;
+        }
+      }
+
+      try {
+        await spawn('npm', ['install', '--save-dev', '@serverless/compose-beta']);
```
[the remark this answers is not in the document]
Comment (pgrzesik) [reply lib/cli/run-compose.js:109]: That's something that I wanted to bring up 💯 I think we should finally get rid of `beta` suffix as I don't imagine we can publish a Framework version with that integration and still use `beta` in package name

[the remark this answers is not in the document]
Comment (mnapoli) [reply lib/cli/run-compose.js:109]: 👍  all good with me to move forward with `@serverless/compose` and publish there.

[the remark this answers is not in the document]
Comment (pgrzesik) [reply lib/cli/run-compose.js:109]: One question - should we just switch the package name to `@serverless/compose` but keep the version as-is? Or just start from scratch? I think keeping the version would make sense and just add a note that prior versions were published as `@serverless/compose-beta`. Does that sound good to you?

[the remark this answers is not in the document]
Comment (mnapoli) [reply lib/cli/run-compose.js:109]: We can keep the versions, I don't think anyone will care about the old ones TBH.

For the release next week we'll want a compose 1.0.0 in any case, so feel free to have a 0.0.x in the meantime.

**Comment by mnapoli** [comment]
Just tried this branch again (removed previous Compose version from the project) and got this:

```
❯ sls-dev deploy

Serverless Compose needs to be installed first. This is a one-time operation.
? Do you want to install Serverless Compose locally with "npm"? Yes
/Users/matthieu/dev/serverless/serverless/bin/serverless.js:79
          throw error;
          ^

TypeError [ERR_INVALID_ARG_TYPE]: The "id" argument must be of type string. Received null
    at new NodeError (node:internal/errors:372:5)
    at validateString (node:internal/validators:120:11)
    at Module.require (node:internal/modules/cjs/loader:992:3)
    at require (node:internal/modules/cjs/helpers:102:18)
    at module.exports (/Users/matthieu/dev/serverless/serverless/lib/cli/run-compose.js:125:5) {
  code: 'ERR_INVALID_ARG_TYPE'
}

Node.js v17.8.0
```

Compose was installed though:

```json
{
  "devDependencies": {
    "@serverless/compose": "^0.0.11"
  }
}
```

**Comment by mnapoli** [comment]
👍  all good

**Pull request merged**